setup.py

```diff
@@ -21,7 +21,7 @@ def find_stub_files(name: str) -> List[str]:
     readme = f.read()
 
 dependencies = [
-    "mypy>=0.980",
+    "mypy>=1.0.0",
```

Comment:
NB! I bumped minimum mypy version to 1.0, since I needed to use `typing_extensions.Self`. I couldn't figure out how to make this work with `_typeshed.Self`.